src/Sulu/Bundle/PageBundle/Preview/PageObjectProvider.php

```diff
@@ -73,6 +74,11 @@ public function setValues($object, $locale, array $data)
         $structure = $object->getStructure();
         foreach ($data as $property => $value) {
             try {
+                if ($property === 'ext') {
```

Comment:
```suggestion
                if ('ext' === $property) {
```

![yoda-giphy](https://github.com/sulu/sulu/assets/1698337/a9a4058a-2b07-458d-a869-104f808239b0)